Index: Makefile
===================================================================
--- Makefile	(revision 14000)
+++ Makefile	(working copy)
@@ -18,7 +18,7 @@
 
 include $(INCLUDE_DIR)/package.mk
 
-define Package/mpc
+define Package/mpc-old
   SECTION:=sound
   CATEGORY:=Sound
   TITLE:=Music Player Daemon Console Client
@@ -25,7 +25,7 @@
   URL:=http://www.musicpd.org/
 endef
 
-define Package/mpc/description
+define Package/mpc-old/description
 	MPD is a music player supporting flac, mp3 and ogg files.
 	It is typically controlled over a network using one of it's many 
 	clients including mpc(console), gmpc(gnome), phpmp(php) etc.
@@ -45,10 +45,10 @@
 )
 endef
 
-define Package/mpc/install
+define Package/mpc-old/install
 	$(INSTALL_DIR) $(1)/usr/bin
 	$(CP) $(PKG_BUILD_DIR)/src/mpc $(1)/usr/bin/
 	$(INSTALL_BIN) ./files/pls-handler.sh $(1)/usr/bin
 endef
 
-$(eval $(call BuildPackage,mpc))
+$(eval $(call BuildPackage,mpc-old))
